bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/SelectedLayersHandler.js

```diff
@@ -54,23 +72,28 @@ class UIService extends StateHandler {
 
     changeOpacity (layer, opacity) {
         this.sandbox.postRequestByName('ChangeMapLayerOpacityRequest', [layer.getId(), opacity]);
-        this.updateState({ layers: this._getLayers() });
+        this.updateLayers();
```

Comment:
This is redundant because LayerViewTabsHandler.js listens for OpacityEvent and refreshes the ui.